packages/components/src/disabled/test/index.js

```diff
@@ -14,7 +14,7 @@ import { Component } from '@wordpress/element';
 import Disabled from '../';
 
 jest.mock( '@wordpress/dom', () => {
-	const focus = require.requireActual( '@wordpress/dom' ).focus;
+	const focus = require.requireActual( '../../../../dom/src' ).focus;
```

Comment:
Can we add `@worpress/dom` back to the jest config?

Reply:
`require.requireActual` is problematic. If you use as `@wordpress/dom`, it will load it from `node_modules` and try to use `main` from `packages.json` - it skips module name mapper (edited). The mocking logic has its own life :slightly_smiling_face: